hadoop-hdds/docs/content/security/SecurityAcls.md

```diff
@@ -63,23 +63,20 @@ volume and keys in a bucket. Please note: Under Ozone, Only admins can create vo
  to the volume and buckets which allow listing of the child objects. Please note: The user and admins can list the volumes owned by the user.
 3. **Delete** – Allows the user to delete a volume, bucket or key.
 4. **Read** – Allows the user to read the metadata of a Volume and Bucket and
-data stream and metadata of a key(object).
+data stream and metadata of a key.
 5. **Write** - Allows the user to write the metadata of a Volume and Bucket and
-allows the user to overwrite an existing ozone key(object).
+allows the user to overwrite an existing ozone key.
 6. **Read_ACL** – Allows a user to read the ACL on a specific object.
 7. **Write_ACL** – Allows a user to write the ACL on a specific object.
 
-<h3>Ozone Native ACL APIs <span class="badge badge-secondary">Work in
-progress</span></h3>
+<h3>Ozone Native ACL APIs</h3>
 
 The ACLs can be manipulated by a set of APIs supported by Ozone. The APIs
 supported are:
 
-1. **SetAcl** – This API will take user principal, the name of the object, type
- of the object and a list of ACLs.
-
-2. **GetAcl** – This API will take the name of an ozone object and type of the
-object and will return a list of ACLs.
-3. **RemoveAcl** - It is possible that we might support an API called RemoveACL
- as a convenience API, but in reality it is just a GetACL followed by SetACL
- with an etag to avoid conflicts.
+1. **SetAcl** – This API will take user principal, the name, type
+of the ozone object and a list of ACLs.
+2. **GetAcl** – This API will take the name and type of the ozone object
+and will return a list of ACLs.
+3. **RemoveAcl** - This API will take the name, type of the
```

Comment:
I think we miss the following here:
 **AddAcl** - This API will take the user principal, name type of ozone object and an ozone ACL, and add it to existing acls of the ozone object.

Reply:
Thanks, will add it.